src/functionalTest/java/uk/gov/hmcts/reform/juddata/cameltest/JrdBatchTestValidationTest.java

```diff
@@ -301,5 +304,24 @@ void testChildSkipsOnParentFailure() throws Exception {
         assertEquals(3, auditList.size()); //Personal, Locations, base-locations only
     }
 
+    @Test
+    void testRowIdInExceptionTable() throws Exception {
+
+        uploadBlobs(jrdBlobSupport, archivalFileNames, true, fileWithEmptyPerIdInAuth);
+        uploadBlobs(jrdBlobSupport, archivalFileNames, false, LeafIntegrationTestSupport.file);
+
+        jobLauncherTestUtils.launchJob();
+        validateExceptionDbRecordCount(jdbcTemplate, exceptionQuery, 2, false);
+        List<Map<String, Object>> exceptionList = jdbcTemplate.queryForList(exceptionQuery);
+
+        List<Long> rowId = exceptionList.stream()
+                .map(i -> i.get("row_id"))
+                .map(j -> (Long) j)
```

Comment:
Instead of performing two transformations, would we be able to do this in a single map? May be like below?
```
List<Long> rowId = exceptionList.stream()
                .map(i -> Long.valueOf(i.get("row_id")))
                .collect(Collectors.toList());
```

Reply:
I am getting the below error
`Cannot resolve method 'valueOf(java.lang.Object)`

Reply:
I guess the below would work:
```
        List<Long> rowId = exceptionList.stream()
                .map(i -> (Long) i.get("row_id"))
                .collect(Collectors.toList());
```

Reply:
did the same just now after your previous comment. thanks